hack/sync_bundle

```diff
@@ -50,14 +50,71 @@ class SyncBundle
     @package = YAML.load(File.read("config/manifests/local-storage-operator.package.yaml"))
   end
 
+  def tag_images(operator_image, diskmaker_image)
+    if operator_image != DEFAULT_OPERATOR_IMAGE
+      check_for_yum_install_dockerfile("./Dockerfile")
+    end
+    if diskmaker_image != DEFAULT_DISKMAKER_IMAGE
+      check_for_yum_install_dockerfile("./Dockerfile.diskmaker")
+    end
+
+    if operator_image != DEFAULT_OPERATOR_IMAGE
+      puts "Building operator image"
+      system("docker build --no-cache -t #{operator_image} -f ./Dockerfile .")
+      system("docker push #{operator_image}")
+    end
+
+    if diskmaker_image != DEFAULT_DISKMAKER_IMAGE
+      puts "Building diskmaker image"
+      system("docker build --no-cache -t #{diskmaker_image} -f ./Dockerfile.diskmaker .")
+      system("docker push #{diskmaker_image}")
+    end
+  end
+
+  def check_for_yum_install_dockerfile(dockerfile)
+    puts "Check for #{dockerfile}"
+    content = File.read(dockerfile)
+    content_lines = content.split("\n")
+    sources = []
+    content_lines.each do |line|
+      if line =~ /^FROM\s+(.+)/
+        sources << $1
+      end
+    end
+
+    sources_are_private = lambda do
+      all_private = true
+      sources.each { |source|
+        if source !~ /^registry/
```

Comment:
nit: won't always hold that a registry is private if it starts with `registry`

Reply:
yes, so if Dockerfile has all `FROM` statements beginning with `registry.blah` - `yum install` will likely fail. It is basically a hack but sort of works in real world usage. :D